.github/dependabot.yml

```diff
@@ -4,3 +4,8 @@ updates:
     directory: "/"
     schedule:
       interval: "monthly"
+    groups:
+      dependencies:
```

Comment:
Same comment as https://github.com/forcedotcom/sf-fx-runtime-java/pull/439/files#r1388586063 :-)

Reply:
added GH action updates and renamed this group to `java-dependencies`